content/blog/open-sourcing-workshops-on-demand-part4-managing-the-backend.md

```diff
@@ -0,0 +1,242 @@
+---
+title: "Open Sourcing Workshops-on-Demand part 4: Manage the Backend"
+date: 2023-07-18T09:28:35.992Z
+author: Frederic Passeron
+authorimage: /img/frederic-passeron-hpedev-192.jpg
+disable: false
+tags:
+  - " opensource"
+  - workshops-on-demand
+---
+I﻿n previous articles of this series dedicated to the [open sourcing of our Workshops-on-Demand project](https://developer.hpe.com/blog/willing-to-build-up-your-own-workshops-on-demand-infrastructure/), I covered the reasons why we open sourced  the project, how we did it. I also explained in details how you could install your own Workshops-on-Demand backend server. I also took the time to detail the automation that was hosted on this backend server. Today, I plan to describe to you the management of this backend server. What is often refered to as day2 operations. 
```

Comment:
change "why we open sourced the project, how we did it." to:
"why we open sourced the project and how we did it."

This is interesting, Fred... right here, I see "I also explained in details how you could..." and I was going to recommend modifying that somewhat, but in the preview, it says "I also took the time to detail the..."

I'm not sure which is the most current. My preference is the "I also took the time to detail the automation..."

The last sentence should be modified to read:
"This is what is often referred to as Day 2 operations." (please note the spelling of "referred")